torchao/prototype/dtypes/bitnet.py

```diff
@@ -0,0 +1,121 @@
+import torch
+from torchao.prototype.dtypes.uint2 import UInt2Tensor, unpack_uint2, pack_uint2
+
+BITNET_OPS_TABLE = {}
+
+def implements(aten_ops):
+    def decorator(fn):
+        for op in aten_ops:
+            BITNET_OPS_TABLE[op] = fn
+        return fn
+    return decorator
+
+def _quantize_int2(x: torch.Tensor, target_dtype: torch.dtype) -> torch.Tensor:
+    # Quantize the input tensor to int2
+    quant = x.sign() + 1
+
+    if target_dtype == torch.uint2:
```

Comment:
I don't think you need torch.uint2 it doesn't do anything. You can remove the 0.3 skip test as well in the test file

Reply:
fixed